packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -213,9 +213,12 @@ export default class MediaEmbedEditing extends Plugin {
 						const id = `fjvideo-${ match[ 2 ] }`;
 
 						return (
-							`<video id=${ id } style="max-width: 100%;" poster="${ posterUrl }" controls controlsList="nodownload">` +
-								`<source type="video/mp4" src=${ url }></source>` +
-							'</video>'
+							'<div style="position: relative; background-color: black; text-align: center;">' +
```

Comment:
Why is the position relative?

Reply:
Not a big reason. Just use position relative as the other elements above.